benchmarking/evaluations/scenarios/project_requirement_preferences_diversify_gender.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+from api.models.enums import DiversifyType, ScenarioAttribute
+from benchmarking.evaluations.enums import PreferenceDirection, PreferenceSubject
+from benchmarking.evaluations.goals import (
+    DiversityGoal,
+    WeightGoal,
+    PreferenceGoal,
+    ProjectRequirementGoal,
+)
+from benchmarking.evaluations.interfaces import Goal
+
+
+class ProjectRequirementPreferenceDiversifyGender:
+    @property
+    def name(self) -> str:
+        return "Meet Project Requirements, Project Preferences, and Diversify Gender"
+
+    @property
+    def goals(self) -> List[Goal]:
+        return [
+            DiversityGoal(DiversifyType.DIVERSIFY, ScenarioAttribute.GENDER.value),
+            ProjectRequirementGoal(),
+            PreferenceGoal(
+                direction=PreferenceDirection.INCLUDE,
+                subject=PreferenceSubject.PROJECTS,
+                max_project_preferences=3,
```

Comment:
this is the kind of scenario that should have an __init__ where this is passed in as a value, since it depends on how you generated the students